lib/SIL/IR/OperandOwnership.cpp

```diff
@@ -278,39 +280,58 @@ OPERAND_OWNERSHIP(EndBorrow, EndBorrow)
 
 #undef OPERAND_OWNERSHIP
 
-// Conditionally either ForwardingConsume or ForwardingBorrow, depending on
-// instruction result's dynamic ownership kind.
-#define FORWARD_ANY_OWNERSHIP(INST)                                            \
-  OperandOwnership                                                             \
-  OperandOwnershipClassifier::visit##INST##Inst(INST##Inst *i) {               \
-    return i->getOwnershipKind().getForwardingOperandOwnership();              \
+// Forwarding operations are conditionally either ForwardingConsumes or
+// ForwardingBorrows, depending on the instruction result's dynamic ownership.
+#define FORWARDING_OWNERSHIP(INST)                                             \
+  OperandOwnership OperandOwnershipClassifier::visit##INST##Inst(              \
+      INST##Inst *i) {                                                         \
+    return i->getOwnershipKind().getForwardingOperandOwnership(false);         \
```

Comment:
Can you put `/*allowsUnowned*/` next to false so we know what the parameter is for?